cmd/start.go

```diff
@@ -29,10 +29,20 @@ func init() {
 }
 
 func runStart(cmd *cobra.Command, args []string) {
+	checkDaemon()
```

Comment:
I think we should also call this on at least a couple other commands, without risking too much overhead: `stop` and  `status` comes to my mind.